apis/v1alpha1/types.go

```diff
@@ -61,6 +60,10 @@ type ProviderConfigSpec struct {
 	// Region for managed resources created using this Alibaba Cloud provider,
 	// e.g. "cn-hangzhou".
 	Region string `json:"region"`
+
+	// Profile is used to extend fields of ProviderConfigSpec.
+	// +kubebuilder:pruning:PreserveUnknownFields
+	Profile map[string]string `json:"profile,omitempty"`
```

Comment:
profile should move to resource spec，because profile is related with a specific resource，and not a global scope